### PR TITLE
Mention "Label Smoothing" in d2l

### PR DESCRIPTION
**Pull request by mstewart141**

### CODE REVIEW

**Comment by mstewart141** [comment]
Hi all, I noticed that d2l did not have any mention of label smoothing, which i've found to be very useful in practice. The book already cites 
```
[3] Szegedy, C., Vanhoucke, V., Ioffe, S., Shlens, J., & Wojna, Z. (2016). Rethinking the inception architecture for computer vision. In Proceedings of the IEEE Conference on Computer Vision and Pattern Recognition (pp. 2818-2826).
```
in one relevant section and I fixed some typos while also highlighting the label smoothing technique advanced therein.

I actually believe it would make sense to dedicate a paragraph somewhere in the text to label smoothing in more detail, but I was not sure where that would belong. I would be happy to take a stab at writing something up if (1) it would be helpful, and (2) someone has a good idea of where it might belong in the text.

File: chapter_convolutional-neural-networks/googlenet.md [hunk 1]
```diff
@@ -36,7 +36,8 @@ class Inception(nn.Block):
         p2 = self.p2_2(self.p2_1(x))
         p3 = self.p3_2(self.p3_1(x))
         p4 = self.p4_2(self.p4_1(x))
-        return nd.concat(p1, p2, p3, p4, dim=1)  # Concatenate the outputs on the channel dimension.
```
Comment (mstewart141) [inline chapter_convolutional-neural-networks/googlenet.md]: previously wrapped very awkwardly in PDF

File: chapter_convolutional-neural-networks/googlenet.md [hunk 2]
```diff
@@ -130,6 +131,7 @@ gb.train_ch5(net, train_iter, test_iter, batch_size, trainer, ctx, num_epochs)
 1. There are several iterations of GoogLeNet. Try to implement and run them. Some of them include the following:
     * Add a batch normalization layer, as described later in this chapter [2].
     * Make adjustments to the Inception block [3].
+    * Use "Label Smoothing" to prevent the net from becoming overconfident in it's predictions [3].
```
Comment (astonzhang) [inline chapter_convolutional-neural-networks/googlenet.md]: Use "Label Smoothing" for model regularization [3].

File: chapter_convolutional-neural-networks/googlenet.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
...
+During the ImageNet Challenge in 2014, a new architecture emerged that outperformed the rest. [Szegedy et al., 2014](https://arxiv.org/abs/1409.4842) proposed a structure that combined the strengths of the NiN and repeated blocks paradigms. At its heart was the rather pragmatic answer to the question as to which size of convolution is ideal for processing. After all, we have a smorgasbord of choices, $1 \times 1$ or $3 \times 3$, $5 \times 5$ or even larger. And it isn't always clear which one is the best. As it turns out, the answer is that a combination of all the above works best. Over the next few years, researchers made several improvements to GoogLeNet. In this section, we will introduce the first version of this model series in a slightly simplified form - we omit the peculiarities that were added to stabilize training, due to the availability of better training algorithms. Notable, the original architecture included several "auxiliary classification" heads which addressed the diminution of gradients in very deep networks; this slightly cumbersome technique has become less popular over time [3].
```
Comment (astonzhang) [inline chapter_convolutional-neural-networks/googlenet.md]: Thanks. It would still be hard for readers without much background to understand the descriptions here: auxiliary classification, diminution of gradients, etc.. Could you revert such changes?

**Comment by astonzhang** [comment]
Could you re-PR by addressing the above comments?

**Comment by astonzhang** [comment]
@mstewart141

Thanks for your contributions to d2l-en (https://github.com/d2l-ai/d2l-en). Do you mind re-licensing all your contributions to us under the new license: MIT-0 for code and CC-BY-SA for text? This will allow more people to use d2l-en for free. All you need to do is replying to this message. Thanks.

Cheers,
Aston

**Comment by mstewart141** [comment]
> @mstewart141

Thanks for your contributions to d2l-en (https://github.com/d2l-ai/d2l-en). Do you mind re-licensing all your contributions to us under the new license: MIT-0 for code and CC-BY-SA for text? This will allow more people to use d2l-en for free. All you need to do is replying to this message. Thanks.

Cheers,
Aston

Yes, no problem, I accept.

**Comment by astonzhang** [comment]
Thanks!
